src/actions/deepLinkActions.js

```diff
@@ -76,29 +59,26 @@ const beginApproveLogin = (query: ApproveLoginQuery) => {
 
 export const executeDeepLinkAction = (deepLink: string) => {
   return async (dispatch: Dispatch) => {
-    const params: Object = url.parse(deepLink, true);
-    if (params.protocol !== 'pillarwallet:') return;
-    switch (params.host) {
+    const { host, protocol, query = {} } = url.parse(deepLink, true) || {};
```

Comment:
I think it would make sense to explicitly handle the case where `url.parse(deepLink, true)` fails.

Maybe something like:

```js
const params = url.parse(deepLink, true);
if (!params) {
  return; // or whatever, but don't continue going
}

const { host, protocol, query = {} } = params;
```

Reply:
Not necessary, but makes sense so added, thanks 👍